MyLibrary/MyLibraryApp.Application/Models/Books/BookResponseModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using MyLibraryApp.DataAccess.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyLibraryApp.Application.Models.Books
+{
+    public class BookResponseModel
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public string Author { get; set; }
```

Comment:
This `Author` property should definitely become a different class & db table and here you should only use its id.

If you don't rely on ids but on strings you can end up in the situation where `Ion Creanga` might appear in your db as `ION Creanga`, `Creanga Ion`, `ION CREANGA`, etc.  That's the pain that relational databases can help with. 😄